eclair-core/src/main/scala/fr/acinq/eclair/Features.scala

```diff
@@ -192,6 +192,11 @@ object Features {
     val mandatory = 50
   }
 
+  case object KeySend extends Feature {
```

Comment:
You should also add this in the `featuresDependency` map.
KeySend depends on `VariableLengthOnion` (you couldn't have known since it's not spec-ed yet!).

Reply:
Done